extensions/twenty/src/services/TwentySDK.ts

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { getPreferenceValues } from "@raycast/api";
+import { Api } from "../enum/api";
+import fetch from "node-fetch";
+import { getActiveDataModelsSchema } from "./zod/schema/dataModelSchema";
+import { dataModelsFieldsSchema } from "./zod/schema/recordFieldSchema";
+
+class TwentySDK {
+  private url!: string;
+  private token!: string;
+
+  constructor() {
+    const { token, url }: { token: string; url: string } = getPreferenceValues();
+    this.token = `Bearer ${token}`;
+    this.url = url ? `${url}/rest` : `https://api.twenty.com/rest`;
```

Comment:
I entered my self-hosted Twenty URL but it crashed. Reason is I copied URL from browser address bar so there was a trailing slash - the URL should account for that :).